src/jsonrpc/miner_jsonrpc_ledger.erl

```diff
@@ -157,7 +157,5 @@ last_challenge(_Height, undefined) -> <<"undefined">>;
 last_challenge(Height, LC) -> Height - LC.
 
 format_ledger_validator(Addr, Val, Ledger, Height, Verbose) ->
-    maps:from_list([
-        {<<"name">>, blockchain_utils:addr2name(Addr)}
-        | blockchain_ledger_validator_v1:print(Val, Height, Verbose, Ledger)
-    ]).
+    InfoPairs = blockchain_ledger_validator_v1:print(Val, Height, Verbose, Ledger),
+    maps:from_list(InfoPairs ++ [ {name, ?BIN_TO_ANIMAL(Addr)}, {address, ?BIN_TO_B58(Addr)} ]).
```

Comment:
So passing in verbose already sets `validator_address` in the returned json. 

Either always pass verbose as false, or drop the `name` entry here?

Reply:
I see. I think that "address" should always be returned and that verbose, if used at all, should cause the name to be returned as well. Currently that is reversed, and frankl, silly.

But there's more work that can be done with this method, actually. I'm noticing that it returns everything as strings, which is silly for a JSON interface since it can return integers and floats, at the very least, natively.

I'm going to remove the :print() can and instead see if I can implement what it does more directly.